static/css/restyle.less

```diff
@@ -702,6 +704,10 @@ button.good {
     color: #FFF;
 }
 
+.hovercard a h3, .secondary li a, #footer a:link, #footer a:visited {
```

Comment:
Where comma separated selectors are used it's easier to read if you add a newline after the comma since scanning shorter lines vertical is easier than parsing the commas in one line horizontally. 

Ideally would be nice to fix that for this whole file at some point.